index.js

```diff
@@ -0,0 +1,9 @@
+const AudioProcessor = require('./lib/utils');
+const DeepgramTranscriber = require('./lib/deepgramTranscriber');
+const DeepgramRedactor = require('./lib/deepgramRedactor');
+
+module.exports = {
```

Comment:
I think it would be simpler (for the caller) if you exported functions for this, rather than Classes. Also, what you should do is export a function that can be called with options, which when invoked returns an object containing functions.  Those functions are then what the calling application uses.  Minimally, the options that can be passed to the function I mentioned above would include a logger object.

Have another look at speech-utils, this is how we do it there.